### DISCUSSION TITLE
Installation URL error

### DISCUSSION THREAD

**Discussion by marcelw60** [lavinir/hassio-onedrive-backup#44] [General]
Hi,  
Whille adding the repository url, I received the following error message:

Cmd('git') failed due to: exit code(128) cmdline: git clone -v --recursive --depth=1 --shallow-submodules -- https://github.com/lavinir/hassio-onedrive-backup /data/addons/git/24c1e4dc stderr: 'Cloning into '/data/addons/git/24c1e4dc'... fatal: unable to access 'https://github.com/lavinir/hassio-onedrive-backup /': URL using bad/illegal format or missing URL '

Any idea where it went wrong?

**Comment by lavinir** [comment]
Hi, no I'm not sure why you would be getting this error.. Maybe some hidden character in the clipboard ? Did you try putting in the address manually (not copy pasting) ?
